client/my-sites/posts/main.jsx

```diff
@@ -56,6 +59,10 @@ const PostsMain = React.createClass( {
 			return false;
 		}
 
+		if ( ! site.jetpack && this.props.author && this.props.myDraftCount === 0 ) {
```

Comment:
While true that `author` will never be anything other than the user in current usage, since it's an ID, seems odd that we make this assumption. Should we instead compare incoming author prop to `getCurrentUserId` in the `connect` selector?